docs/source/regular_grid.rst

```diff
@@ -10,10 +10,10 @@ RegularGrid
     :nosignatures:
 
     RegularGrid
+    RegularGrid.def
```

Comment:
This is not needed. Or at least we don't use it in the other files. I am not sure about the generated outcome...

Reply:
I am not sure either. The file was generated automatically when ran `make_rst.sh`. Some other things seem to be fishy, I am not sure how to fix that. E.g. `EZyRB/docs/build/html/code.html` lists the new module on the left under **Code Documentation** and on the right, but the class members are missing on the right. Also, when I click on any other module (e.g. **Code documentation -> linear**), the new module disappears on the left under **Code Documentation**. When I click on **Code documentation -> regular_grid** i.e. `EZyRB/docs/build/html/regular_grid.html`, the whole left side menu appears to be collapsed.
I dont really know if that is only on my end or if these are problems for other users too? Do you have a suggestion how to fix that? Maybe @ndem0 knows?
Another thing is that there seem to be two .rst files for the same module, `podae.rst` and `pod_ae.rst`. One appears to be autogenerated, the other one hand written. Quite a few of the .rst files seem to be different (outdated?) from what is generated when i run `make_rst.sh`. Maybe it is something for a new issue?

Reply:
We are not using that script anymore! Sorry, it's not written, I gonna remove it in the next release of EZyRB (#232).
The file now looks correct, thanks!